videointelligence/video_analyze/gen/template.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+//+build ignore //# omit
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	video "cloud.google.com/go/videointelligence/apiv1"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+
+	"github.com/golang/protobuf/ptypes"
+	"golang.org/x/net/context"
+)
+
+//# if gcs
+//# replace __SUFFIX__( URI(
+//# end
+//# if !gcs
+//# replace __SUFFIX__( (
+//# end
+func boilerplate() { //# omit
+	//# def dorequest
+	ctx := context.Background()
+	client, err := video.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+	//# if !gcs
+
+	f, err := os.Open(file)
```

Comment:
We could replace the 5 lines below with:
```go
fileBytes, err := ioutil.ReadFile(file)
```

Reply:
Great call. Done.